lib/core/src/Cardano/Wallet/CoinSelection/Fee.hs

```diff
@@ -285,178 +320,3 @@ splitChange = go
             if isValidCoin newChange
             then newChange : go newRemaining as
             else a : go rest as
-
--- | A linear equation on the transaction size. Represents the @\s -> a + b*s@
--- function where @s@ is the transaction size in bytes, @a@ and @b@ are
--- constant coefficients.
-data TxSizeLinear =
-    TxSizeLinear (Quantity "lovelace" Double) (Quantity "lovelace/byte" Double)
```

Comment:
ok - I see this has just been moved -  but I am still qurious about it (question above)